contracts/staking/test/migration.ts

```diff
@@ -131,9 +127,7 @@ blockchainTests('Migration tests', env => {
                 expect(params[4]).to.bignumber.eq(stakingConstants.DEFAULT_PARAMS.cobbDouglasAlphaNumerator);
                 expect(params[5]).to.bignumber.eq(stakingConstants.DEFAULT_PARAMS.cobbDouglasAlphaDenominator);
                 expect(params[6]).to.eq(wethProxyAddress);
-                expect(params[7]).to.eq(ethVaultAddress);
-                expect(params[8]).to.eq(rewardVaultAddress);
-                expect(params[9]).to.eq(zrxVaultAddress);
+                expect(params[7]).to.eq(zrxVaultAddress);
```

Comment:
I'm going to leave these for now because of some quirks with some params being numbers/BigNumbers (the event types won't match whatever wrapper we write), and we're not doing this _too_ frequently.

Reply:
I think @mzhu25  is on to something. I added it to the backlog.